ext/gvl_tracing_native_extension/gvl_tracing.c

```diff
@@ -75,25 +76,16 @@ static inline void initialize_thread_id(void) {
 }
 
 static inline void render_thread_metadata(void) {
```

Comment:
Looking at the perfetto rendering for the `thread_name_with_extension.rb` example...

![image](https://user-images.githubusercontent.com/2785847/234893397-1b008d69-deaf-4404-9563-eb3e6e5e442b.png)

it always prints the `tid` anyway, so it gets duplicated now. Previously this was the thread serial which was different :)

So I'd suggest a slight tweak here -- printing only the name.